scripts/gh-projects/examples/matchline/create-issues.sh

```
@@ -0,0 +1,268 @@
+ensure_label "needs-external-review" "ededed" "PR exceeds external-review threshold"
+
+# -----------------------------------------------------------------------------
+# Phase 0 — Foundations & access
+# -----------------------------------------------------------------------------
+P0_URL=$(create_parent "Phase 0: Foundations & access" \
+  "$SCRIPT_DIR/phase-0/parent.md" "matchline,phase-0")
+P0_NUM="${P0_URL##*/}"
+echo "P0 PARENT: $P0_URL"
+
+F=$(prep_body "$SCRIPT_DIR/phase-0/c1.md" "$P0_NUM")
+read P0_C1_URL P0_C1_NUM _ <<<"$(create_child "Fix P0: invalid Anthropic Haiku model ID in functions/src/llm/config.ts" \
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)